### ISSUE TITLE
Crash Report

### ISSUE THREAD

**Issue by Tyler000** [ranger/ranger#1253]
#### Runtime Environment

- Operating system and version: Arch Linux kernel: 4.17.14
- Terminal emulator and version: termite v13
- Python version: 3.7
- Ranger version/commit: ranger master 1.9.1
- Locale: en_US.UTF-8

#### Current Behavior
Randomly crashed while browsing, works fine when started again though.

#### Expected Behavior
To not crash when browsing files.
#### Context

I was simply browsing files and it crashed. Ranger worked fine after I restarted it. Just wanted to submit the traceback if it could help at all.
#### Possible Solutions
No idea...
#### Steps to reproduce
1. Open ranger.
2. Browse files.
3. Randomly crash after looking in a few different files of different type. Mostly jpg, txt and pdf with all the appropriate recommended programs installed for displaying those.

#### Traceback

```
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: en_US.UTF-8
Current file: '/home/tyler/Documents/Misc/NVCC/trans_credit.pdf'

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 236, in _generate_lines
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/view_miller.py", line 100, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 176, in draw
    self._draw_file()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 203, in _draw_file
    Pager.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 100, in draw
    for line, i in zip(line_gen, range(self.hei)):
RuntimeError: generator raised StopIteration

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues

```

**Comment by Gomme-** [comment]
Looks like the issue I have reported. See #1251

**Comment by mxovd** [comment]
This is already fixed in  #1244 , you can clone the latest git

**Comment by toonn** [comment]
Yep, breaking changes in python 3.7, either wait for the next release, run the version on master or downgrade python.
